### ISSUE TITLE
Can we start using R5 version 

### ISSUE THREAD

**Issue by Sameer2810** [microsoft/fhir-server#3368]
I am submitting ImagingStudy Resource of version 5 but it is still giving below error

 "issue": [
        {
            "severity": "error",
            "code": "invalid",
            "diagnostics": "Error occurred when parsing model: 'Type checking the data: Encountered unknown element 'partOf' at location 'Resource.partOf[0]' while parsing'."
        }
    ]

I am running R5.Web project but I think it is still using R4 firely version. Is there any change required so that I can use R5 resource

**Comment by feordin** [comment]
We do have an R5 project, but it is still an older release of R5. There is a PR in progress to move the R5 project to the release version of R5. We will get it updated soon.

**Comment by Sameer2810** [comment]
When we accept R5 release?

**Comment by EXPEkesheth** [comment]
@Sameer2810 - We do not have defined timeline yet. I have added this to our backlog to adopt R5 firely version.

**Comment by EXPEkesheth** [comment]
@Sameer2810 - is there a specific scenario or use-case for which you need R5 support and cannot be addressed as part of R4?

**Comment by aematteo** [comment]
Is there a release date of R5 stable version? The actually MS FHIR R5 is based on R5 5.0.0-snapshot-1 and some fields are missing in these datamodels. For example Organization.description is missing.

**Comment by EXPEkesheth** [comment]
@aematteo - There is no release date defined for R5 support in FHIR service. Once we have concrete timelines, we will respond to this thread.

**Comment by ZuSe** [comment]
Any news on it?
Would love to use R5 for a project related to the EU EHRxF (compare https://github.com/hl7-eu/laboratory).

Right now only HAPI/Smile seem to provide decent support of R5.

**Comment by EXPEkesheth** [comment]
@ZuSe are you using FHIR service in OSS or as managed service?

**Comment by cacao-matcha** [comment]
Any news on an updated R5? The version still is 5.0.0 snapshot1. I have run into the same issue as above with the location resource.

**Comment by LuckyNumber0111** [comment]
Yes, please keep the FHIR Store up to date. Else the Azure Health Data Services make no sense at all and cannot be used for projects
